cmd/gpu_plugin/rm/gpu_plugin_resource_manager.go

```diff
@@ -16,14 +16,22 @@ package rm
 
 import (
 	"context"
-	"math/rand"
```

Comment:
this is added by the previous commit so can we make that to use `crypto/rand` directly?

Reply:
yep, will change.

The `crypto/rand` change was due to lint warnings which I noticed yesterday.